integration_test/workflow_tests/run_tests.py

```diff
@@ -0,0 +1,34 @@
+import unittest
+
+import testtools
+
+import test_packages
+
+
+class TracingStreamResult(testtools.StreamResult):
+    def __init__(self):
+        self.test_results = {}
+
+    def status(self, *args, **kwargs):
+        self.test_results[kwargs["test_id"]] = kwargs["test_status"]
+
+
+def run_tests():
+    suite = unittest.TestLoader().loadTestsFromModule(test_packages)
+    concurrent_suite = \
+        testtools.ConcurrentStreamTestSuite(lambda: ((case, None) for case in suite))
+    result = TracingStreamResult()  # testtools.StreamResult()
+    result.startTestRun()
+    concurrent_suite.run(result)
+    result.stopTestRun()
+
+    print(f"Checking results on {len(result.test_results)} tests...")
```

Comment:
Instead of print statements, I wonder if we shouldn't utilize the logging libraries to give us some better flexibility and provide a consistent format that we could easily parse later.

Reply:
https://bugs.earthdata.nasa.gov/browse/ORCA-513

Reply:
Switched to logging statements. Leaving card as grouping/labeling of logs would be nice.